lib/example_apply_response.json

```diff
@@ -0,0 +1,218 @@
+        "safeguarding_concerns": false
+      },
+      {
+        "id": 9351,
+        "name": "Max Schroeder",
+        "email": "410beb42e8@example.com",
+        "relationship": "Odit facere omnis. Dicta vitae quam. Perspiciatis quod ea.",
+        "reference": "You are awesome",
+        "referee_type": "academic",
+        "safeguarding_concerns": false
+      }
+    ],
+    "qualifications": {
+      "gcses": [
+        {
+          "id": 13777,
+          "qualification_type": "gcse",
+          "non_uk_qualification_type": null,
+          "subject": "science",
+          "grade": "A (Predicted)",
+          "start_year": "2021",
+          "award_year": "1968",
+          "institution_details": "Mayert Institute, GB",
+          "awarding_body": "Brighthurst Secondary College",
+          "equivalency_details": "Omnis vitae voluptas. Earum cupiditate ipsam. Maxime esse quo. Incidunt dolor dolore. Dolorum impedit deserunt. Neque consequatur saepe. Atque qui ut. Inventore fugit consectetur. Facere maiores odio.",
+          "hesa_degtype": null,
+          "hesa_degsbj": null,
+          "hesa_degclss": null,
+          "hesa_degest": null,
+          "hesa_degctry": null,
+          "hesa_degstdt": null,
+          "hesa_degenddt": null
+        },
+        {
+          "id": 13776,
+          "qualification_type": "gcse",
+          "non_uk_qualification_type": null,
+          "subject": "english",
+          "grade": "B",
+          "start_year": "2021",
+          "award_year": "2016",
+          "institution_details": "Mosciski College, GB",
+          "awarding_body": "Mallowpond Secondary College",
+          "equivalency_details": "Rerum eum nihil. Autem quis non. Sit eos ducimus. Accusamus quod eius. Doloribus eos sint. Quam id perferendis. Et voluptatum est. Quia optio officia. Fugiat voluptatem nihil. Voluptate est amet. Vol.",
+          "hesa_degtype": null,
+          "hesa_degsbj": null,
+          "hesa_degclss": null,
+          "hesa_degest": null,
+          "hesa_degctry": null,
+          "hesa_degstdt": null,
+          "hesa_degenddt": null
+        },
+        {
+          "id": 13775,
+          "qualification_type": "gcse",
+          "non_uk_qualification_type": null,
+          "subject": "maths",
+          "grade": "C (Predicted)",
+          "start_year": "2021",
+          "award_year": "1996",
+          "institution_details": "Southern Friesen Academy, GB",
+          "awarding_body": "Mallowpond High",
+          "equivalency_details": "Voluptates sint nemo. Enim sed asperiores. Enim dolores natus. Consequuntur nesciunt soluta. Velit ducimus ea. Officiis voluptatem temporibus. Rerum repellendus quas. Distinctio asperiores dolorem. S.",
+          "hesa_degtype": null,
+          "hesa_degsbj": null,
+          "hesa_degclss": null,
+          "hesa_degest": null,
+          "hesa_degctry": null,
+          "hesa_degstdt": null,
+          "hesa_degenddt": null
+        }
+      ],
+      "degrees": [
```

Comment:
looks like the same/similar degrees as ours

Reply:
ahhh degrees! i knew there was something else I wanted to check - thanks!